src/cleanvision/imagelab.py

```diff
@@ -21,29 +24,82 @@
 from cleanvision.utils.utils import get_filepaths, deep_update_dict
 from cleanvision.utils.viz_manager import VizManager
 
+__all__ = ["Imagelab"]
+
 OBJECT_FILENAME = "imagelab.pkl"
 TImagelab = TypeVar("TImagelab", bound="Imagelab")
 
 
 class Imagelab:
+    """A single class to find all types of issues in image datasets.
+
+    Parameters
+    ----------
+    data_path : str
+        Path to image files. Imagelab will recursively retrieve all image files from the specified path
+
+    Examples
+    --------
+
+    Basic usage of Imagelab class
+
+    .. code-block:: python
+        from cleanvision.imagelab import Imagelab
+        imagelab = Imagelab(data_path)
+        imagelab.find_issues()
+        imagelab.report()
+
+
+    Attributes
```

Comment:
@sanjanag @elisno 
Currently Imagelab.issues contains column called: `<issue_type>_bool` for the hard decision on whether image contains issue or not. But Datalab.issues contains column called: `is_<issue_type>_issue` ?

We should unify these. I prefer the latter sort of naming convention where it is more clear what this column means just from the dataframe itself without user having to refer to the documentation.

Reply:
Renamed